sympy/printing/pretty/pretty.py

```diff
@@ -2284,6 +2285,10 @@ def pretty_print(expr, **settings):
     use_unicode : bool or None, optional
         use unicode characters, such as the Greek letter pi instead of
         the string pi.
+    root_notation : bool, optional
+        use root form or fractional form for expressions having rational
+        roots of the form 1/n. Set to False for fractional form;
```

Comment:
"expressions having rational roots" means something different. Also, "or" is typically not used when describing a boolean parameter; for example, "use_unicode" says "use unicode characters, such as the Greek letter pi". It is understood that False means: don't do that thing.

I suggest "use root form for powers with exponent of the form 1/n. Default is True."